addons/survey/static/tests/components/question_page_one2many_field_tests.js

```diff
@@ -301,4 +304,59 @@ QUnit.module("QuestionPageOneToManyField", (hooks) => {
             assert.strictEqual(target.querySelector(".o_is_section [name=title]").innerText, "a");
         }
     );
+
+    QUnit.test("full page list question creation", async (assert) => {
+        const indexes = [];
+        const questions = [];
+        for (let i = 1; i <= 45; i++) {
+            questions.push({
+                id: i,
+                is_page: false,
+                title: `question ${i}`,
+                sequence: i,
+            });
+            indexes.push(i);
+        }
+
+        console.log(questions);
+        serverData.models.survey_question.records = questions;
+        serverData.models.survey.records[0].question_and_page_ids = indexes;
```

Comment:
Maybe instead of using, indexes, you could do:
```python
serverData.models.survey.records[0].question_and_page_ids = questions.map((q) => q.id);
```